benchmarks/quad_hexagon.py

```diff
@@ -27,9 +27,9 @@ def time_open_dataset(self):
         """Time to open a `UxDataset`"""
         ux.open_dataset(grid_path, data_path)
 
-    def mem_open_dataset(self):
-        """Memory occupied by a `UxDataset`"""
-        return ux.open_dataset(grid_path, data_path)
+    # def mem_open_dataset(self):
```

Comment:
commented for a reason?
same for others.

Reply:
This benchmark was giving me a lot of trouble. Regardless of what changes I made to the environment or configuration, it would always throw and error. 

This may be something to do with what is written here:

https://asv.readthedocs.io/en/stable/writing_benchmarks.html#memory

However, this benchmark works fine for our other workflow.